datamade/la-metro-councilmatic
# add Fernando Dutra image and bio

## Overview

This PR adds Fernando Dutra's headshot and biography. I made these changes by trying out instructions from #705 

### Checklist



## Testing Instructions

 * check `councilmatic/settings_jurisdiction.py` to make sure Fernando Dutra's image and bio appear in `MANUAL_HEADSHOTS` and `MEMBER_BIOS`, respectively.
 * make sure the slug for Fernando Dutra is the slug from the [production site](https://boardagendas.metro.net/person/fernando-dutra-19bb1bb05d11/)

Handles #707
